routing/pathfind_test.go

```diff
@@ -150,28 +149,15 @@ type testChan struct {
 }
 
 // makeTestGraph creates a new instance of a channeldb.ChannelGraph for testing
-// purposes. A callback which cleans up the created temporary directories is
-// also returned and intended to be executed after the test completes.
-func makeTestGraph(useCache bool) (*channeldb.ChannelGraph, kvdb.Backend,
-	func(), error) {
-
-	// First, create a temporary directory to be used for the duration of
-	// this test.
-	tempDirName, err := ioutil.TempDir("", "channeldb")
+// purposes.
+func makeTestGraph(t *testing.T, useCache bool) (*channeldb.ChannelGraph, kvdb.Backend, error) {
```

Comment:
nit: line length here and with two more modified function definitions.

Reply:
Resolved in 889bcd4727c6600cafd9bf9b8994db2a2c68951f